components/layouts/partials/navigation.js

```diff
@@ -26,6 +28,8 @@ const List = styled.ul`
 
 export default function Navigation({ closeNavigation }) {
     const auth = useAuth();
+    const matrix = useMatrix(auth.getAuthenticationProvider('matrix'));
```

Comment:
Since https://github.com/medienhaus/medienhaus-spaces/pull/44 was merged it's enough to say `useMatrix()` -- there's no parameter that the function accepts anymore 👍